src/content/docs/en/guides/assets.mdx

```diff
@@ -0,0 +1,213 @@
+
+Remote images are also valid, but are not automatically optimized by Astro's build process.
+
+```md
+<!-- src/pages/post-1.md -->
+# My Markdown Page
+
+<!-- Local image stored at src/assets/stars.png -->
+![A starry night sky.](../../assets/stars.png)
+<img src="/_assets/stars.hash.png" alt="A starry night sky.">
+
+<!-- Remote image on another server -->
+![Astro](https://astro.build/assets/logo.png)
+<img src="https://astro.build/assets/logo.png" width="25" alt="Astro">
+```
+
+### Content Collections
+
+A common use case for blog posts is to reference assets in their metadata (think, cover image for an article, picture for an author etc).
+
+With the `src/assets` folder, we're also introducing a new `asset` helper to the `astro:content` module. The `asset` helper will validate that the user references a valid asset from the `src/assets` folder:
+
+```ts
+import { asset, defineCollection, z } from "astro:content";
+
+const blogCollection = defineCollection({
+  schema: z.object({
+    title: z.string(),
+    image: asset({ width: 1120, height: 1120 }),
+  }),
+});
+```
+
+Assets referred this way will be transformed automatically to the same shape as if the image was imported, so they're ready to be passed to the `Image` component or the `getImage` function.
+
+## Features
+
+Enabling asset support allows you to access the `astro:assets` module. This module exposes the following features:
+
+- `Image` (see [Image Component](#image-component))
+
+### Image Component
+
+The `<Image>` component generates an `<img>` tag with all the necessary attributes needed to display the image while avoiding Content Layout Shift **(CLS)**.
+
+:::tip[What is Content Layout Shift?]
+CLS is a score that reflects how much content shifted on your page during loading. The ideal number for CLS is as close to zero as possible.
+:::
+
+```astro
+---
+import { Image } from 'astro:assets';
+import myImage from "../assets/my_image.png"; // Image is 1600x900
+---
+
+<!-- `alt` is mandatory on the Image component -->
+<Image src={myImage} alt="..." />
+```
+
+```astro
+<!-- Output -->
+<!-- Image is optimized, proper attributes are enforced -->
+<img
+  src="/_astro/my_image.hash.webp"
+  width="1600"
+  height="900"
+  decoding="async"
+  loading="lazy"
+  alt="..."
+/>
+```
+
+### Properties
+
+#### width and height
+
+These properties are used to size the image while ensuring there will be no CLS. These properties are required for remote images. When using local images, the `width` and `height` can be automatically inferred from the source file.
+
+#### format
+
+`format` is used to set the [image file type](https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Image_types#common_image_file_types). This property is optional, but the default is `webp`.
+
+#### quality
+
+`quality` affects the image transformation quality. It can either be a preset from `low`, `medium`, `high`, `max` or a number from `0` to `100`. This property is optional.
```

Comment:
We can add something here like "Each image file type handles the `quality` value differently.`

Reply:
```suggestion
`quality` affects the image transformation quality. It can either be a preset from `low`, `medium`, `high`, `max` (automatically normalized between formats) or a number from `0` to `100` (interpreted differently between formats.) This property is optional.
```